Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -26,9 +26,7 @@
 
 GCC_PREFIX_HEADER = WebCorePrefix.h;
 
-WK_USE_APPLEJPEGXL = $(WK_USE_APPLEJPEGXL_$(WK_PLATFORM_NAME));
-WK_USE_APPLEJPEGXL_macosx = $(WK_USE_APPLEJPEGXL$(WK_MACOS_1400));
-WK_USE_APPLEJPEGXL_MACOS_BEFORE_1400 = YES;
+WK_USE_APPLEJPEGXL = NO;
```

Comment:
Won't this turn JPEGXL off for all builds? Is this mean to go to a branch only?

Reply:
@emw-apple Should this be something like:

```
WK_USE_APPLEJPEGXL = $(WK_USE_APPLEJPEGXL_$(WK_PLATFORM_NAME));
WK_USE_APPLEJPEGXL_macosx = $(WK_USE_APPLEJPEGXL$(WK_MACOS_1400));
WK_USE_APPLEJPEGXL_MACOS_SINCE_1400 = YES;
WK_USE_APPLEJPEGXL_MACOS_BEFORE_1400 = NO;
```

Reply:
Pretty sure the current patch is correct (we don't want to use AppleJPEGXL on Sonoma, see https://bugs.webkit.org/show_bug.cgi?id=257768)

Reply:
If Tim's satisfied, so am I. Adding merge-queue so we can move on!